db-engine/src/main/java/com/splicemachine/db/iapi/util/DecimalUtil.java

```diff
@@ -64,6 +65,7 @@ static DataValueDescriptor getDecimalDb2(DataValueDescriptor dvd,
             break;
             case DECFLOAT: {
                 bigDecimal = SQLDecfloat.getBigDecimal(dvd);
+                roundingMode = BigDecimal.ROUND_HALF_EVEN; // should be configurable (DB-10672)
```

Comment:
How was this rounding mode picked?  Normal rounding (as done in math class, or in spark) uses ROUND_HALF_UP I believe.

Reply:
Found out in the standup ROUND_HALF_EVEN is the DB2 default.